packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -37,107 +37,28 @@ const StyledTableRow = (c: any) =>
     transientOptions
   )(
     ({
-      $hover,
-      $selected,
-      $expanded,
       $striped,
-      $variantList,
-      $variantListHead,
-      $stripedColor,
-      $type,
+      $stripedColorEven,
+      $stripedColorOdd,
     }: {
-      $hover: boolean;
-      $selected: boolean;
-      $expanded: boolean;
       $striped: boolean;
-      $variantList: boolean;
-      $variantListHead: boolean;
-      $type: string;
-      $stripedColor: string;
+      $stripedColorEven: string;
+      $stripedColorOdd: string;
     }) => ({
-      backgroundColor: theme.table.rowBackgroundColor,
-      color: "inherit",
-      verticalAlign: "middle",
-      outline: 0,
-      minHeight: 32,
-      "tr&": {
-        height: 32,
-      },
-
-      ":hover": {
-        ...($type === "body" && {
-          backgroundColor: theme.table.rowHoverColor,
-        }),
-      },
-      ...($hover && {
-        transition: "background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms",
-        "&:hover": {
-          backgroundColor: theme.table.rowHoverColor,
-        },
-      }),
-      ...($selected && {
-        backgroundColor: theme.table.selectedRowBackgroundColor,
-      }),
-      ...($expanded && {
-        "& > *[role=cell]": {
-          borderBottom: "none",
-        },
-        [`&.${tableRowClasses.expanded}`]: {
-          backgroundColor: theme.colors.atmo1,
-        },
-      }),
       ...($striped && {
-        "&:nth-of-type(even)": {
-          backgroundColor: $stripedColor,
+        [`&:nth-of-type(even)`]: {
```

Comment:
We're not using variables, so this doesn't have to be computed. ESLint (sometimes) complains about this

```suggestion
        "&:nth-of-type(even)": {
```